boards/shields/st7735r_generic/Kconfig.defconfig

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_ST7735R_ADA_160x128
+
+if DISPLAY
+
+config SPI
+	default y
+
+config ST7735R
+	default y
+
+choice ST7735R_PIXEL_FORMAT
+	default ST7735R_RGB565 if SHIELD_ST7735R_ADA_160x128
+endchoice
+
+if LVGL
+
+config LVGL_DISPLAY_DEV_NAME
+	default "ST7735R"
+
+config LVGL_HOR_RES_MAX
+	default 160
+
+config LVGL_VER_RES_MAX
+	default 128
+
+config LVGL_BITS_PER_PIXEL
+	default 16 if SHIELD_ST7735R_ADA_160x128
```

Comment:
Either missing a default or the `if SHIELD_ST7735R_ADA_160x128` is unnecessary

Reply:
Fixed